easybuild/easyconfigs/c/CIRCexplorer2/CIRCexplorer2-2.3.8-foss-2020b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonPackage'
+
+name = 'CIRCexplorer2'
+version = '2.3.8'
+
+homepage = 'http://circexplorer2.readthedocs.io/'
+description = "CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['8261ee884f6d895131ce681d5eda6554376e973851a9de4dc50e2b3e242ccc80']
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    # sticking to v2.1.0, since using v2.1.1 has problems, cfr. https://www.biostars.org/p/186979/
+    ('TopHat', '2.1.2'),
```

Comment:
Trying with TopHat v2.1.2, but may need to fall back to using v2.1.0 instead, see also https://github.com/easybuilders/easybuild-easyconfigs/pull/6554#discussion_r211962344

Reply:
follow up in #12857 ...